src/app/file/file-select/file-tree.component.ts

```diff
@@ -42,8 +42,10 @@ export class FileTreeComponent implements OnInit, OnDestroy {
         this.unsubscribe.complete();
     }
 
-    onChildTreeClick(value: string) {
-        this.select.emit(value);
+    onChildTreeClick(path: string) {
+        const finalPath = path.replace('/user/', '');
```

Comment:
We should probably use 'user' as a constant so that we can replace it by something else (like 'Home') if needed